pages/api/user/create.ts

```diff
@@ -25,7 +25,7 @@ export default async function create(
       }
     });
 
-    const playerUrl = `${process.env.BASE_URL}${playerData.tournamentId}/users/${user.id}`;
+    const playerUrl = `${process.env.BASE_URL}tournaments/${playerData.tournamentId}/users/${user.id}`;
```

Comment:
Olis sinänsä johdonmukaisempaa, jos BASE_URL olisi vain hostname ilman kauttaviivaa, mutta tällä hetkellä mentäneen näin.

Reply:
Ah, asiaan saatiinkin oitis muutos.